tables_app/src/main/java/org/opendatakit/tables/fragments/TableManagerFragment.java

```diff
@@ -142,6 +150,18 @@ protected void updateTableIdList() {
         }
       }
     }
+    if(fragSortOrder != null)
+    {
+      Collections.sort(tableNameStructs, new Comparator<TableNameStruct>() {
+        @Override
+        public int compare(TableNameStruct o1, TableNameStruct o2)
+        {
+          return o1.getLocalizedDisplayName().compareTo(o2.getLocalizedDisplayName());
+        }
+      });
+      if(fragSortOrder == Constants.TABLE_SORT_ORDER.SORT_DESC)
+        Collections.reverse(tableNameStructs);
```

Comment:
Instead of reversing the list, it would be better if the sort order is taken into account in the Comparator.